services/dkp-insights/1.0.0/helmrelease/helmrelease.yaml

```diff
@@ -21,14 +21,7 @@ spec:
   upgrade:
     remediation:
       retries: 30
-      # After the database migration has happened, the previous version of
```

Comment:
@asekretenko I think our tooling strips out comments, did we want to persist these comments here?